Master-Game/Entities/Enemy.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace MasterGame.Entities
+{
+    public class Enemy : HumanEntity
```

Comment:
When this is created, the base is a human entity. This means that the enemy will be a human type. Is this something that we want? Or do we want to create an enemy entity?